### PR TITLE
PPF-407 -  Contact tab: add validation on presence of first name, last name and e-mail when adding a contact

### PR DESCRIPTION
**Pull request by brampauwelyn**
### Added
- [new post route: storeContact](https://github.com/cultuurnet/publiq-platform/pull/1017/commits/f5a2a9376bf7ef68b97cc08874829ea0fdcb3f7a)
- [Add StoreContactRequest](https://github.com/cultuurnet/publiq-platform/pull/1017/commits/fb90e0d0a41ae277287e4c7bf950fae58965515b)

### Changed
- [Only close modal when the request is successful](https://github.com/cultuurnet/publiq-platform/pull/1017/commits/55037a59ec76fb7ea50842854e5d896b0e2d45d7)
- [Use new post endpoint for creation of new contributors](https://github.com/cultuurnet/publiq-platform/pull/1017/commits/a8ce609fbd9a69ebddc2f99a9723333319d6783a)

### Removed
- [newContributor info from UpdateContactInfoRequest](https://github.com/cultuurnet/publiq-platform/pull/1017/commits/51b477ccd58ebf778b2749db1b4a7fb5c30ec583)

Screenshot 2024-04-16 at 11 16 10

---
Ticket: https://jira.uitdatabank.be/browse/PPF-407

### CODE REVIEW

File: app/Domain/Integrations/Controllers/IntegrationController.php [hunk 1]
```diff
@@ -229,6 +231,15 @@ public function updateContacts(string $id, UpdateContactInfoRequest $request): R
         return Redirect::back();
     }
 
+    public function storeContact(StoreContactRequest $request, string $id): RedirectResponse
```
Comment (LucWollants) [inline app/Domain/Integrations/Controllers/IntegrationController.php:234]: Maybe best to write also a unit test for this new endpoint. It is important to test the happy path and the edge case of an unauthorized user.

Comment (simon-debruijn) [reply app/Domain/Integrations/Controllers/IntegrationController.php:238]: Done

[Add tests for storeContact method](https://github.com/cultuurnet/publiq-platform/pull/1017/commits/2e124236085e70bd541f1003a142dd87950f91a5)

**Review by Anahkiasen — APPROVED** [review]
